### PR TITLE
Add horizontal scrollbar into the large table inside description

### PR DESCRIPTION
**Pull request by luka-nextcloud**
Signed-off-by: Luka Trovic <luka@nextcloud.com>

* Resolves: #2642
* Target version: master 

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: src/components/card/Description.vue [hunk 1]
```diff
@@ -262,6 +262,8 @@ export default {
 
 #description-preview {
 	min-height: 100px;
+	width: max-content;
+	overflow-x: scroll;
```
Comment (juliushaertl) [inline src/components/card/Description.vue:266]: Haven't tried out yet, but would `auto` also work maybe to not always have a scorllbar?

Comment (juliushaertl) [inline src/components/card/Description.vue:266]:
```suggestion
	overflow-x: auto;
```

Let's drop the max-content as that would also avoid long lines to break - and the auto seems to work fine to not always have a scrollbar in some browsers.

**Review by juliushaertl — CHANGES_REQUESTED** [review]
See inline 😉

**Review by eneiluj — APPROVED** [review]
:+1:

**Review by mejo- — APPROVED** [review]

**Pull request merged**